dev-libs/libxml2/libxml2-2.12.0.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Note: Please bump in sync with dev-libs/libxslt
```

Comment:
Did you see this?

Reply:
Ah, you did, but it's a separate PR. Please do them both in one (separate commits).